src/main/scala/scala/tools/refactoring/analysis/CompilationUnitDependencies.scala

```diff
@@ -481,7 +478,7 @@ trait CompilationUnitDependencies extends CompilerApiExtensions with ScalaVersio
 
     val deps = result.values.toList
 
-    deps.filterNot(_.symbol.isPackage).toList
+    deps.filterNot(_.symbol.hasPackageFlag).toList
```

Comment:
I'm surprised that `hasPackageFlag` compiles on 2.10. I thought the deprecation stuff only points to things that were introduced in 2.11.